cmd/openshift-install/waitfor.go

```diff
@@ -46,6 +46,9 @@ func newWaitForBootstrapCompleteCmd() *cobra.Command {
 
 			err = waitForBootstrapComplete(ctx, config, rootOpts.dir)
 			if err != nil {
+				logrus.Info("Use the following commands to gather logs from the cluster")
```

Comment:
this should point users to run the `openshift-install --dir <root-dir> gather bootstrap` command.
and not the `ssh scp commands.`

Reply:
Modified, ptal

Reply:
Modified again to drop `--help` and set `--dir` in #1800.